src/actions/organizationActions.js

```diff
@@ -14,6 +14,8 @@ export function organizationSelect(orgId) {
   return function(dispatch) {
     dispatch(push('/'));
 
+    localStorage.setItem('app.selectedOrganization', orgId);
```

Comment:
perhaps we can make a method in localStorageUtils for this as well.

Reply:
mmm... Why should we do that?
Are you thinking in adding any check in the wrapper method for setItem?

Reply:
Just to keep all the localStorage related things together. 

Another bonus is that then the `app.selectedOrganization` string gets defined in only one place, because there is another 

` localStorage.setItem('app.selectedOrganization', selectedOrganization);`

further down in this file.